utils/lib/Quickstart.ts

```diff
@@ -42,36 +42,57 @@ const SUPPORT_LEVEL_ENUMS: SupportLevelMap = {
 type ComponentType = typeof Alert | typeof Dashboard | typeof DataSource;
 type Components = InstanceType<ComponentType>;
 
+enum ConfigKey {
+  DataSource = 'dataSourceIds',
+  AlertPolicies = 'alertPolicies',
+  Dashboards = 'dashboards', 
+};
+
 interface ConfigToMutationMap {
-  configKey: string;
+  configKey: ConfigKey;
   mutationKey: string;
   ctor: ComponentType;
-}
+};
 
 const ConfigToMutation: ConfigToMutationMap[] = [
-  { configKey: 'alertPolicies', mutationKey: 'alertConditions', ctor: Alert },
-  { configKey: 'dashboards', mutationKey: 'dashboards', ctor: Dashboard },
   {
-    configKey: 'dataSourceIds',
+    configKey: ConfigKey.AlertPolicies,
+    mutationKey: 'alertConditions',
+    ctor: Alert,
+  },
+  {
+    configKey: ConfigKey.Dashboards,
+    mutationKey: 'dashboards',
+    ctor: Dashboard,
+  },
+  {
+    configKey: ConfigKey.DataSource,
     mutationKey: 'dataSourceIds',
     ctor: DataSource,
   },
 ];
 
+export interface QuickstartContext {
```

Comment:
Maybe a comment here acknowledging the way we're handling core data sources. That way we have context during the next change.

Reply:
https://github.com/newrelic/newrelic-quickstarts/pull/1649/commits/966191cc9c4f208ad3adf5eb4ad8086e81501a93